src/strongbox/core.clj

```diff
@@ -1145,14 +1149,14 @@
 
 (defn-spec remove-addon nil?
   "removes the given addon. if addon is part of a group, all addons in group are removed"
-  [toc ::sp/toc]
+  [toc :addon/toc]
```

Comment:
review this

Reply:
`toc` is simply the minimum viable addon. You can delete addons that haven't been matched against the database